apps/codeforafrica/src/pages/opportunities/[slug].page.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 import ArticlePage from "@/codeforafrica/components/ArticlePage";
 import Page from "@/codeforafrica/components/Page";
 import { getPageStaticProps } from "@/codeforafrica/lib";
+import { getAllPostsWithSlug } from "@/codeforafrica/lib/api";
```

Comment:
Instead of including CMS-specific code in pages, we can defined `getPageStaticPath` in lib/index.js (similar to getPageStaticProps). This way pages don't need to be know which CMS back-end a certain piece of content comes from.